pages/cloud-providers.html

```diff
@@ -28,33 +28,18 @@
 
         <div class="card flow-small">
             <div class="card__title">
-                Chère de Prince
+                La Bécasse
             </div>
 
             <p class="text-secondary">
                 French, free
             </p>
 
             <p>
-                <strong><a href="https://cheredeprince.net/">cheredeprince.net</a></strong>
+                <strong><a href="https://rss.cheredeprince.net/">cheredeprince.net</a></strong>
```

Comment:
I think we should keep the landing page, and potentially also indicate the direct link. Otherwise there is the reverse problem, namely that the user lands on a FreshRSS instance without knowing who operates it, why, the conditions, etc.

Reply:
I agree, my intention was to conduct people to the global presentation of each provider so they could understand where they're going and who is going to manage their data. It helps to not consider the providers as some impersonal structure.

Reply:
> I think we should keep the landing page, and potentially also indicate the direct link. Otherwise there is the reverse problem, namely that the user lands on a FreshRSS instance without knowing who operates it, why, the conditions, etc.

How does this look for a landing page? Within the card-info itself:
`        <div class="card flow-small">
            <div class="card__title">
                <strong><a href="https://rss.cheredeprince.net/">Chère de Prince</a></strong>
            </div>`